core/arch/arm/include/mm/core_mmu.h

```diff
@@ -464,6 +467,9 @@ static inline TEE_Result cache_op_outer(enum cache_op op __unused,
 /* Check cpu mmu enabled or not */
 bool cpu_mmu_enabled(void);
 
+/* Check if platform define nsec DDR range(s) */
+bool core_mmu_is_nsec_ddr_configured(void);
```

Comment:
Renaming: I've understood `core_mmu_nsec_ddr_is_configured()` is better than the interrogative way. I even think `core_mmu_nsec_ddr_is_defined()` fits better.